nixos/tests/nixos-test-driver/ocr.py

```diff
@@ -0,0 +1,4 @@
+machine.wait_for_x()
+machine.succeed("xterm >&2 &")
+machine.wait_for_text("root@", timeout=10)
+# assert "root@" in machine.get_screen_text_variants()
```

Comment:
Why is this commented out?

Reply:
Because it doesn't _quite_ work this way, I'll edit it to a variant that does work